src/sentry/static/sentry/app/views/organizationDiscover/data.jsx

```diff
@@ -6,29 +6,29 @@ const TYPES = {
 };
 
 export const PROMOTED_TAGS = [
-  {name: 'tags[level]', type: TYPES.STRING},
-  {name: 'tags[logger]', type: TYPES.STRING},
-  {name: 'tags[server_name]', type: TYPES.STRING},
-  {name: 'tags[transaction]', type: TYPES.STRING},
-  {name: 'tags[environment]', type: TYPES.STRING},
-  {name: 'tags[site]', type: TYPES.STRING},
-  {name: 'tags[url]', type: TYPES.STRING},
-  {name: 'tags[app_device]', type: TYPES.STRING},
-  {name: 'tags[device]', type: TYPES.STRING},
-  {name: 'tags[device_family]', type: TYPES.STRING},
-  {name: 'tags[runtime]', type: TYPES.STRING},
-  {name: 'tags[runtime_name]', type: TYPES.STRING},
-  {name: 'tags[browser]', type: TYPES.STRING},
-  {name: 'tags[browser_name]', type: TYPES.STRING},
-  {name: 'tags[os]', type: TYPES.STRING},
-  {name: 'tags[os_name]', type: TYPES.STRING},
-  {name: 'tags[os_rooted]', type: TYPES.BOOLEAN},
-  {name: 'tags[sentry:release]', type: TYPES.STRING},
+  {name: 'level', type: TYPES.STRING},
+  {name: 'logger', type: TYPES.STRING},
+  {name: 'server_name', type: TYPES.STRING},
+  {name: 'transaction', type: TYPES.STRING},
+  {name: 'environment', type: TYPES.STRING},
+  {name: 'site', type: TYPES.STRING},
+  {name: 'url', type: TYPES.STRING},
+  {name: 'app_device', type: TYPES.STRING},
+  {name: 'device', type: TYPES.STRING},
+  {name: 'device_family', type: TYPES.STRING},
+  {name: 'runtime', type: TYPES.STRING},
+  {name: 'runtime_name', type: TYPES.STRING},
+  {name: 'browser', type: TYPES.STRING},
+  {name: 'browser_name', type: TYPES.STRING},
+  {name: 'os', type: TYPES.STRING},
+  {name: 'os_name', type: TYPES.STRING},
+  {name: 'os_rooted', type: TYPES.BOOLEAN},
+  {name: 'sentry:release', type: TYPES.STRING},
 ];
 
 // All tags are assumed to be strings, except the following
 export const SPECIAL_TAGS = {
-  'tags[os_rooted]': TYPES.BOOLEAN,
+  os_rooted: TYPES.BOOLEAN,
```

Comment:
If `PROMOTED_TAGS` contains the type information, I don't think we need a separate map here enumerating the ones that aren't strings?

Reply:
Good point, removed all the types on the promoted tag list (it's only loaded when tags query fails)